### DISCUSSION TITLE
What is the necessary changes to go from karavan version 3.20.1 to the latest version

### DISCUSSION THREAD

**Discussion by Elias-Swais** [apache/camel-karavan#874] [Q&A] [answered]
im just wondering what do i need to change in my karavan application to divert from version 3.20.1 to the latest version, should i update the quarkus as well or should i just change the dependancy to the latest version?

**Comment by mgubaidullin** [comment]
Do you use vs-code extension or web-app?

**Reply by Elias-Swais** [reply]
Web app

**Reply by mgubaidullin** [reply, marked as answer]
To upgrade Karavan,Just install new version and point to the same git repository.

To update Camel in the projects. Set required version in the project application.properties
